docs/topics/asset/giveaway/single.md

```diff
@@ -0,0 +1,75 @@
+# Asset Single Giveaway Smart Contract
+
+## Overview
+
+Admin locks asset with merkle root and user can claim the asset with matching params.
+
+Similarly to Land presales, we have an off-chain merkle tree.
+This time the key is not the land id but the user’s ethereum address.
+
+Once a user claim its tokens, it cannot claim again
+
+On the frontend, we can show the list of assets the user entitled to. Once the user clicks claim, the frontend can fetch from the backend the merkle proof and any extra parameter to perform the tx call.
```

Comment:
-the user **is** entitled to.

-"Once the user claims the asset" sounds better to me, what do you think?

-any extra parameter needed to perform the tx call